corehq/apps/analytics/tasks.py

```diff
@@ -231,6 +231,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user['email']
+        if not email:
+            continue
```

Comment:
The `get_email` call isn't actually used in this code path.  The empty emails are being returned from an Elastic query, and there only 8 out of about 4000 that were empty.